src/Fields/interpolate.jl

```diff
@@ -132,16 +134,16 @@ ZRegGrid = Union{ZRegularRG, ZRegularLLG, ZRegOrthogonalSphericalShellGrid}
 
 @inline function fractional_z_index(z::FT, locs, grid::ZRegGrid) where FT
     z₀ = znode(1, 1, 1, grid, locs...)
-    Δz = @inbounds first(zspacings(grid, locs...))
-    return convert(FT, (z - z₀) / Δz)
+    Δz = Δz(1, 1, 1, grid, locs...)
```

Comment:
I think we should use the pointwise operators defined in the operator module instead of the `zspacing` function and so on. (there is a conflict in the name here though, I ll correct it)